src/java.base/share/classes/jdk/internal/module/Modules.java

```diff
@@ -155,9 +153,7 @@ public static void addUses(Module m, Class<?> service) {
     public static void addProvides(Module m, Class<?> service, Class<?> impl) {
         ModuleLayer layer = m.getLayer();
 
-        PrivilegedAction<ClassLoader> pa = m::getClassLoader;
-        @SuppressWarnings("removal")
-        ClassLoader loader = AccessController.doPrivileged(pa);
+        ClassLoader loader = m.getClassLoader();
 
         ClassLoader platformClassLoader = ClassLoaders.platformClassLoader();
```

Comment:
You can remove the blank line at L157 now, it was only there to make the doPriv use clearer.

Reply:
Removed blank line